modules/outbound-command-event-handler/test/integration/application/outbound_command_event_handler.test.ts

```diff
@@ -269,11 +271,9 @@ describe("Tests for Outbound Command Event Handler", () => {
     expect(filteredEvents.length).toBe(2);
     // Check the data contents for domain event
     expect(filteredEvents[0].getName()).toBe('PartyInfoRequestedDmEvt');
-    expect(JSON.parse(JSON.stringify(filteredEvents[0].getContent())).path).not.toContain('undefined');
```

Comment:
I originally had this undefined as the content had a field called path, that had /parties/MSISDN/<MSISDNID>/undefined. This test is to make sure the word "undefined" does not appear in the field value. @kleyow

Reply:
To my knowledge atm Sri, the content in this message contains no `path` field.